public/js/login.js

```diff
@@ -0,0 +1,37 @@
+const email = document.getElementById('email');
+const password = document.getElementById('password');
+const btnLogin = document.getElementById('btn_login');
+const msg = document.getElementById('msg');
+
+btnLogin.addEventListener('click', (e) => {
+  e.preventDefault();
+  const emailV = email.value.trim();
```

Comment:
more descriptive names : 
```suggestion
  const emailValue = email.value.trim();
```